docs/getting-started/integration.md

```diff
@@ -11,26 +11,76 @@ Chart.js can be integrated with plain JavaScript or with different module loader
 </script>
 ```
 
-## Webpack
+## Common JS
 
 ```javascript
-import Chart from 'chart.js';
+var Chart = require('chart.js');
 var myChart = new Chart(ctx, {...});
 ```
 
-## Common JS
+## Bundlers (Webpack, Rollup, etc.)
 
 ```javascript
-var Chart = require('chart.js');
+import Chart from 'chart.js';
 var myChart = new Chart(ctx, {...});
 ```
 
+**Note:** Moment.js is installed along Chart.js as dependency. If you don't want to use Momemt.js (either because you use a different date adapter or simply because don't need time functionalities), you will have to configure your bundler to exclude this dependency (e.g. using [`externals` for Webpack](https://webpack.js.org/configuration/externals/) or [`external` for Rollup](https://rollupjs.org/guide/en#peer-dependencies)).
+
+```javascript
+// Webpack
+{
+    externals: {
+        moment: 'moment'
+    }
+}
+```
+
+```javascript
+// Rollup
+{
+    external: {
+        ['moment']
+    }
+}
+```
+
 ## Require JS
 
+**Important:** RequireJS [can **not** load CommonJS module as is](https://requirejs.org/docs/commonjs.html#intro), so be sure to require one of the UMD builds instead (i.e. `dist/Chart.js`, `dist/Chart.min.js`, etc.).
+
 ```javascript
-require(['path/to/chartjs/dist/Chart.js'], function(Chart) {
+require(['path/to/chartjs/dist/Chart.min.js'], function(Chart){
     var myChart = new Chart(ctx, {...});
 });
 ```
 
-> **Important:** RequireJS [can **not** load CommonJS module as is](https://requirejs.org/docs/commonjs.html#intro), so be sure to require one of the built UMD files instead (i.e. `dist/Chart.js`, `dist/Chart.min.js`, etc.).
+**Note:** starting v2.8, Moment.js is now an optional dependency for `Chart.js` and `Chart.min.js`. That means you need to make sure Moment.js is fully loaded **before** requiring Chart.js. You can either use a shim:
```

Comment:
Instead of "That means you need to make sure Moment.js is fully loaded" should we say "If you want to use the time scale, that means you need to make sure your date library is fully loaded"

Reply:
Not really, it's just about Moment.js, not other date libs.

Reply:
What about:

> In order to use the time scale with Moment.js, you need to make sure Moment.js is fully loaded **before** requiring Chart.js

Reply:
That sounds fine to me